src/main/resources/db_scripts/create_external_accounts_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE external_accounts (
+   user_id INTEGER NOT NULL,
+   provider_name TEXT NOT NULL,
+   provider_user_identifier TEXT,
+   provider_last_updated TIMESTAMP WITHOUT TIME ZONE,
+   CONSTRAINT external_accounts_pk PRIMARY KEY (user_id, provider_name),
+   CONSTRAINT external_accounts_fk FOREIGN KEY (user_id) REFERENCES public.users (id)
+       ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+-- Add stub records to prevent synchronisation?
```

Comment:
Shoudl this just be a config value to turn sync on or off globally or is it important we can sync certain records and not others?

Reply:
This was mostly intended as a bit of a hack so we could test it with just staff users first; we will want to sync some accounts and not others and this uses the functionality built into the sync process for deleted or invalid accounts to avoid being synced.